website/docs/r/privatelink_endpoint_serverless.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: privatelink_endpoint_serverless"
+sidebar_current: "docs-mongodbatlas-datasource-privatelink-endpoint-serverless"
+description: |-
+Describes a Serverless PrivateLink Endpoint
+---
+
+
+# Data Source: privatelink_endpoint_serverless
+
+`privatelink_endpoint_serverless` Provides a Serverless PrivateLink Endpoint resource.
+This is the first of two resources required to configure PrivateLink for Serverless, the second is [mongodbatlas_privatelink_endpoint_service_serverless](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service_serverless).
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
```

Comment:
```suggestion
-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
-> **NOTE:** Feature unavailable in Free and Shared-Tier Clusters
This feature is not available for M0 free clusters, M2, and M5 clusters. To learn more about which features are unavailable, see [Atlas M0 (Free Cluster), M2, and M5 Limitations.](https://www.mongodb.com/docs/atlas/reference/free-shared-limitations/#std-label-atlas-free-tier)
```

Reply:
mainly since we call this out our Atlas Admin API/UI Set Up page for this resource: https://www.mongodb.com/docs/atlas/security-serverless-private-endpoint/

Reply:
mainly since we do this Set Up page for Atlas Admin API/UI: https://www.mongodb.com/docs/atlas/security-serverless-private-endpoint/

Reply:
@martinstibbe delete this one